### DISCUSSION TITLE
I am reviving the plugin!

### DISCUSSION THREAD

**Discussion by 05TEVE** [iRayanKhan/homebridge-tuya#434] [New device help]
Hi guys, I am working to get a new version published soon. I have pushed a beta version now but i am keen to understand what PRs I should prioritise merging in next :)

**Comment by metaljay** [comment]
Ace, have you thought about implementing the latest developments around ‘cloud cutter’ 

https://github.com/tuya-cloudcutter/tuya-cloudcutter/wiki/FAQ#can-i-uncut-a-device-and-connect-it-back-to-tuya

**Comment by pespinel** [comment]
> Hi guys, I am working to get a new version published soon. I have pushed a beta version now but i am keen to understand what PRs I should prioritise merging in next :)

The first thing you should do is this: https://github.com/iRayanKhan/homebridge-tuya/discussions/490
